json_serializable/pubspec.yaml

```diff
@@ -29,6 +29,10 @@ dependencies:
   source_gen: ^1.3.2
   source_helper: ^1.3.0
 
+dependency_overrides:
+  json_annotation:
+    path: ../json_annotation
+
```

Comment:
I wasn't sure if the override had its place in production, since it isn't currently, but hey it looks like the only thing I missed was 5 minutes of official documentation

Reply:
it NEEDS to be here until we publish json_annotation

I'll take care of it